src/components/tasks/TasksContent.tsx

```diff
@@ -109,6 +110,11 @@ export const TasksContent = ({ dev }: { dev: boolean }) => {
         }
     }, [tasksData.tasks]);
 
+    const loadingRef = useRef<HTMLDivElement | null>(null);
```

Comment:
to avoid `| null` the `useRef` can be defined like below
```
import {ElementRef} from 'react'

const loadingRef = useRef<ElementRef<"div">>(nulll);
const bottomBoundaryRef = useRef<ElementRef<"div">>(null);
```

FYI, your type definition is also fine, this is just a suggestion.

Reply:
Done